include/wil/resource.h

```diff
@@ -4233,6 +4233,40 @@ namespace wil
 #if !defined(NOWINABLE)
     typedef unique_any<HWINEVENTHOOK, decltype(&::UnhookWinEvent), ::UnhookWinEvent> unique_hwineventhook;
 #endif
+#if !defined(NOCLIPBOARD)

```

Comment:
This seems like a good fit for the "call method on close" helper, `wil::unique_call` plus a method, like:

```c++
using unique_clipboard = unique_call<decltype(::CloseClipboard()), &::CloseClipboard>;

unique_clipboard open_clipboard(HWND hwnd) {
    return { OpenClipboard(hwnd) != FALSE };
}
```

Reply:
I updated the code to use the `unique_call` type.